<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>621</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>选择规则</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="choseLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择规则：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ruleLable">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>示例规则*</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="rulesTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>741</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>规则</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>模式</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>概览</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>男子一千米</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>较小值优先</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>111111111111111111111111111111111111111111111111111</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>男子五十米</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>较小值优先</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>~</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>男子实心球</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>较小值优先</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>~</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>男子篮球绕杆</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>较大值优先</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>~</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>530</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creatButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导入方案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导出方案</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
